examples/fluent-bit/amazon-managed-service-for-prometheus/README.md

```diff
@@ -0,0 +1,53 @@
+## FireLens Example: Prometheus forward metrics - using the Fluent Bit image with init tag
+
+This example shows how to forward metrics with Fluent Bit's prometheus remote write output plugin to an Amazon Managed Service for Prometheus workspace.
+
+### Step 1: Create config file locally
+
+**fluent-bit.conf**
+
+```
+# FireLens Example: Prometheus forward metrics - using the Fluent Bit image with init tag
+
+# Scape node metrics every 20 seconds (use any metrics input plugin)
+[INPUT]
+    Name                        node_exporter_metrics
+    Tag                         node_metrics
+    Scrape_interval             20
+
+# Send metrics to AMP via Remote Write
+[OUTPUT]
+    Name prometheus_remote_write
+    Match node_metrics
+    Host aps-workspaces.us-west-2.amazonaws.com
+    Port 443
+    Uri /workspaces/< my-amp-workspace-id >/api/v1/remote_write
+    AWS_Auth On
+    AWS_region us-west-2
+    Tls On
+    Tls.verify On
+    add_label app my-ecs-app
+    add_label color blue
+```
+
+**Note:** you can find this config file in the `config-files` directory of this example, please modify according to the actual situation to match your needs.
```

Comment:
> actual situation

your resource values? can this language be more technical sounding?

Reply:
Updated wording